### ISSUE TITLE
Custom template support

### ISSUE THREAD

**Issue by brandonb927** [asciidisco/grunt-patternprimer#7]
Now, before I get flamed and responded with "RTFM!!", having Handlebars or Mustache template support would be really cool. I'm looking into implementing it, but was wondering if this was in the works already?

**Comment by asciidisco** [comment]
Not in the works (and by the way, I was & never will be one of those RTFM guys).
Can you give me some examples on what do you mean with template support.

What are the benefits; Does it make things possible/easier without raising the complexity; etc.

**Comment by brandonb927** [comment]
Well, I ask because I was previously in the works of implementing our product style guide for front-end devs, but i found that this project (specifically the template HTML structure, two-thirds/one-third widths) was slightly too opinionated to be useful (to me at least) but upon looking through the source, I realized it would be a good candidate for an external template engine to be used. I haven't come up with anything but I might hack on it this week and see if something comes of it.

**Comment by stevebritton** [comment]
@brandonb927 I've actually thought about incorporating the grunt-assemble module into the project and think it would fit nicely.

**Comment by brandonb927** [comment]
@stevebritton ohhh that actually looks really cool

**Comment by stevebritton** [comment]
Yeah, I use the grunt-assemble module for a bunch of projects that have static content, so it would be great for this setup.

**Comment by asciidisco** [comment]
Sounds like a good idea, I would like to keep this plugin as simple (and close to the original impl.) as possible, but go ahead; if it gets too "sophisticated" we can simply publish the fork or make it available in addition to this "original" lib.
